### DISCUSSION TITLE
Feature request: Button to "pin" a window (always on top)

### DISCUSSION THREAD

**Discussion by redsolver** [nextapps-de/winbox#128] [Ideas]
Basically have a little pin icon which when pressed causes the window to always be on top of all other windows. This is useful for pop-out video players or audio controls for example.

**Comment by quacksire** [comment]
You could just use a browser with PiP (Picture in Picture)

**Comment by kodie** [comment]
You could easily accomplish this with CSS.

```css
.winbox.pinned {
  z-index: 9999 !important;
}
```

```js
new WinBox("Pinned Window", { class: ['pinned'] })
```

**Comment by ts-thomas** [comment]
Adding custom controls to the window toolbar is now supported. You can combine this with the workaround above.
